### PR TITLE
test,Makefile: Add more test information in README.md

### PR DESCRIPTION
**Pull request by GMHDBJD**
### What is changed and how it works?
- add more test information in tests/README.md 
- copy test file in compatibility_test in Makefile

### Check List 

Tests 
 - No code

### CODE REVIEW

File: tests/README.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
 
 ## Preparations
 
-1. The following executables must be copied or generated or linked into these locations, `mydumper` and `sync_diff_inspector` can be downloaded from [tidb-enterprise-tools-latest-linux-amd64](http://download.pingcap.org/tidb-enterprise-tools-latest-linux-amd64.tar.gz):
+1. The following executables must be copied or generated or linked into these locations, `mydumper` and `sync_diff_inspector` can be downloaded from [tidb-enterprise-tools-latest-linux-amd64](http://download.pingcap.org/tidb-enterprise-tools-latest-linux-amd64.tar.gz), `tidb-server` can be downloaded from [tidb-master-linux-amd64](https://download.pingcap.org/tidb-master-linux-amd64.tar.gz):
```
Comment (csuzhangxc) [inline tests/README.md:4]: In DM 2.0, we do not need `mydumper` now, how about removing it from this README and do not check where it exsits? cc @lichunzhu 

(but when cherry-pick to 1.0, we still needed it).

File: tests/README.md [hunk 2]
```diff
@@ -34,7 +34,9 @@
 
 1. Run `make dm_integration_test_build` to generate DM related binary for integration test
 
-2. Run `make integration_test` to execute the integration tests. This command will
+2. Setup two MySQL servers with binlog enabled first and set `GTID_MODE=ON`, export proper environment variable `MYSQL_HOST1`, `MYSQL_PORT1`, `MYSQL_PASSWORD1` and `MYSQL_HOST2`, `MYSQL_PORT2`, `MYSQL_PASSWORD2`, default is `127.0.0.1`, `3306`, `123456` and `127.0.0.1`, `3307`, `123456`.
```
Comment (csuzhangxc) [inline tests/README.md:37]: How about using a _table_ format to show these variables and their default value (ditto for L58)?

File: tests/README.md (hunk 1 above)
Comment (lichunzhu) [reply tests/README.md:4]: Yes, I think we should remove it for master, v2.0 but kept it for v1.0.

Comment (GMHDBJD) [reply tests/README.md:4]: Shall we remove `mydumper-path` config in tests/*/dm-task.yaml ?

Comment (csuzhangxc) [reply tests/README.md:4]: Yes, you can remove them.

Comment (GMHDBJD) [reply tests/README.md:4]: How about remove them in another pr?

Comment (csuzhangxc) [reply tests/README.md:4]: OK

File: tests/README.md [hunk 3]
```diff
@@ -34,7 +34,12 @@
 
 1. Run `make dm_integration_test_build` to generate DM related binary for integration test
 
-2. Setup two MySQL servers with binlog enabled first and set `GTID_MODE=ON`, export proper environment variable `MYSQL_HOST1`, `MYSQL_PORT1`, `MYSQL_PASSWORD1` and `MYSQL_HOST2`, `MYSQL_PORT2`, `MYSQL_PASSWORD2`, default is `127.0.0.1`, `3306`, `123456` and `127.0.0.1`, `3307`, `123456`.
+2. Setup two MySQL servers with binlog enabled first and set `GTID_MODE=ON`, export proper environment variable. Variable and its default value showed in following table.
```
Comment (csuzhangxc) [inline tests/README.md:37]: I think `variables` and `are` should be used. 😂

**Pull request merged**